### PR TITLE
Loop-loop 2D EM inversion

### PR DESCRIPTION
**Pull request by lheagy**
Generate an example where we invert secondary magnetic field data from a single frequency loop-loop EM survey. We use a horizontal co-planar orientation with three different source-receiver offsets. 

The forward modelling is performed in 3D, but the inversion model is 2D. 

image

This example was inspired from http://onlinelibrary.wiley.com/doi/10.1002/2013WR014864/full.  

The notebook that will be built in the documentation is here: 
[plot_loop_loop_2Dinversion.ipynb.zip](https://github.com/simpeg/simpeg/files/1778268/plot_loop_loop_2Dinversion.ipynb.zip) 
and the documentation of that example is here: 
[2D inversion of Loop-Loop EM Data — SimPEG 0.7.2 documentation.pdf](https://github.com/simpeg/simpeg/files/1778761/2D.inversion.of.Loop-Loop.EM.Data.SimPEG.0.7.2.documentation.pdf)

@sgkang, @rowanc1: would you mind taking a look? cc @dougoldenburg, if you have a chance to take a look, any input is appreciated!

### CODE REVIEW

File: examples/07-fdem/plot_loop_loop_2Dinversion.py [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+# vertically. We estimate the trade-off parameter, beta, between the data
+# misfit and regularization by the largest eigenvalue of the data misfit and
+# the regularization.
+
+dmisfit = DataMisfit.l2_DataMisfit(survey)
+reg = Regularization.Simple(inversion_mesh)
+opt = Optimization.InexactGaussNewton(maxIterCG=10, maxIter=10, remember="xc")
+invProb = InvProblem.BaseInvProblem(dmisfit, reg, opt)
+
+betaest = Directives.BetaEstimate_ByEig(beta0_ratio=0.25)
```
Comment (sgkang) [inline examples/07-fdem/plot_loop_loop_2Dinversion.py]: Any reason why you are fixing beta?

File: examples/07-fdem/plot_loop_loop_2Dinversion.py [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+# -----
+#
+# Define the survey and model parameters
+#
+
+sigma_surface = 10e-3
+sigma_deep = 40e-3
+sigma_air = 1e-8
+
+coil_separations = [0.32, 0.71, 1.18]
```
[the remark this answers is not in the document]
Comment (lheagy) [reply examples/07-fdem/plot_loop_loop_2Dinversion.py]: Hi Seogi, yeah, I wouldn't be surprised... I went for the larger cells simply to have it run quickly. Do you have a screen-shot? We can update it if you think that is best

File: examples/07-fdem/plot_loop_loop_2Dinversion.py [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+from SimPEG.Utils import mkvc
+
+###############################################################################
+# Setup
+# -----
+#
+# Define the survey and model parameters
+#
+
+sigma_surface = 10e-3
```
[the remark this answers is not in the document]
Comment (lheagy) [reply examples/07-fdem/plot_loop_loop_2Dinversion.py]: I based it on the DC inversion result from: http://onlinelibrary.wiley.com/doi/10.1002/2013WR014864/abstract

image

File: examples/07-fdem/plot_loop_loop_2Dinversion.py (hunk 1 above)
Comment (lheagy) [reply examples/07-fdem/plot_loop_loop_2Dinversion.py]: Just for simplicity

**Pull request merged**